assets/python/ml/pyml:post_processing:pca_2d_clusters:matplotlib.j2.py

```diff
@@ -0,0 +1,68 @@
+# ----------------------------------------------------------------- #
+#                                                                   #
+#   ROC Curve Generator                                             #
+#                                                                   #
+#   Computes and displays the Receiver Operating Characteristic     #
+#   (ROC) curve. This is restricted to binary classification tasks. #
+#                                                                   #
+# ----------------------------------------------------------------- #
```

Comment:
Docstring needs to be changed

Reply:
Docstring has been changed, and updated with a summary of how the plot is generated